src/app/content/search/components/SearchResultsSidebar/index.spec.tsx

```diff
@@ -195,7 +195,7 @@ describe('SearchResultsSidebar', () => {
 
     jest.useFakeTimers();
     if (sidebar.searchSidebar.current) {
-      sidebar.searchSidebar.current.dispatchEvent(animationEvent());
+      sidebar.searchSidebar.current.dispatchEvent(animationEvent() as any);
```

Comment:
why is the any needed for this?

Reply:
It is because `animationEvent` is returning `CustomEvent` instead of `Event`. I'll update the function instead using `any`
```
Argument of type 'CustomEvent<unknown>' is not assignable to parameter of type 'Event'.
  Types of property 'srcElement' are incompatible.
    Type 'EventTarget | null' is not assignable to type 'Element | null'.
          Type 'EventTarget' is missing the following properties from type 'Element': assignedSlot, attributes, classList, className, and 119 more.ts(2345)
    ```